lib/datasets/house-of-representative.rb

```diff
@@ -0,0 +1,114 @@
+                        :progress_url,
+                        :text,
+                        :text_url,
+                        :bill_type,
+                        :submitter,
+                        :submitter_in_house_groups,
+                        :house_of_representatives_of_accepted_bill_on_preliminary_consideration,
+                        :house_of_representatives_of_preliminary_refer_on,
+                        :house_of_representatives_of_preliminary_refer_commission,
+                        :house_of_representatives_of_accepted_bill_on,
+                        :house_of_representatives_of_refer_on,
+                        :house_of_representatives_of_refer_commission,
+                        :house_of_representatives_of_finished_consideration_on,
+                        :house_of_representatives_of_consideration_result,
+                        :house_of_representatives_of_finished_deliberation_on,
+                        :house_of_representatives_of_deliberation_result,
+                        :house_of_representatives_of_attitude_of_in_house_group_during_deliberation,
+                        :house_of_representatives_of_support_in_house_group_during_deliberation,
+                        :house_of_representatives_of_opposition_in_house_group_during_deliberation,
+                        :house_of_councillors_of_accepted_bill_on_preliminary_consideration,
+                        :house_of_councillors_of_preliminary_refer_on,
+                        :house_of_councillors_of_preliminary_refer_commission,
+                        :house_of_councillors_of_accepted_bill_on,
+                        :house_of_councillors_of_refer_on,
+                        :house_of_councillors_of_refer_commission,
+                        :house_of_councillors_of_finished_consideration_on,
+                        :house_of_councillors_of_consideration_result,
+                        :house_of_councillors_of_finished_deliberation_on,
+                        :house_of_councillors_of_deliberation_result,
+                        :promulgated_on,
+                        :law_number,
+                        :submitters,
+                        :supporters_of_submitted_bill)
+
+    SPLIT_COLUMN_CHAR_ON_HEADER = "／".freeze
+    SPLIT_COLUMN_CHAR_ON_FIELD = SPLIT_COLUMN_CHAR_ON_HEADER
+
+    def initialize
+      super()
+
+      @metadata.id = "house-of-representative"
+      @metadata.name = "Bill of the House of Representatives of Japan"
+      @metadata.url = "https://smartnews-smri.github.io/house-of-representatives"
+      @metadata.licenses = ["MIT"]
+      @metadata.description = "Bill of the House of Representatives of Japan"
+    end
+
+    def each
+      return to_enum(__method__) unless block_given?
+
+      open_data do |csv|
+        csv.each do |row|
+          row = split_csv_column(row)
+          row.fields.each_with_index do |field, idx|
+            row[idx] = JapaneseDateParser.new(field).parse
+          end
```

Comment:
Creating a new `JapaneseDateParser` object for each parsing may be slow.
How about passing the input to `#parse` instead of `#initialize`?

```ruby
parse = JapaneseDateParser.new
row.fields.each_with_index do |field, idx|
  row[idx] = parser.parse(field)
end
```

Reply:
I see. Thanks for your suggestion.
Added at 3134f12, creating a new `JapaneseDateParser` object once!